ext/exif/exif.c

```diff
@@ -2467,6 +2463,7 @@ static void add_assoc_image_info(zval *value, int sub_array, image_info_type *im
 		image_info_value *info_value = &info_data->value;
 		const char *name = info_data->name;
 		if (!name) {
+			char uname[64];
```

Comment:
This introduces a scope use after free because name will be set to uname but uname goes out of scope

Reply:
Indeed, fixed.